.idea/runConfigurations/Debug_errorprones.xml

```diff
@@ -0,0 +1,15 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Is this intentional? I've generally seen run configurations be added by leveraging the gradle plugin

Reply:
Do you mean that we can add some code in RemoteDebugJavaCompilePlugin to create this configuration file in the repo we are running in, e.g. gradle-baseline, if it doesn't exist yet?